src/coprocessor/endpoint.rs

```diff
@@ -811,7 +811,7 @@ impl<E: Engine> Endpoint<E> {
 }
 
 fn make_error_batch_response(batch_resp: &mut coppb::StoreBatchTaskResponse, e: Error) {
-    warn!(
+    debug!(
```

Comment:
Unlike key-is-locked error, the other types of log maybe helpful and unusual. Can we only remove warn log for key-is-locked error only(maybe region error as well), but keep it for `Error::DeadlineExceeded` and `Error::MaxPendingTasksExceeded`.

Reply:
The printing logic is updated, PTAL

Reply:
@you06 Friendly ping, PTAL